sdks/java/extensions/sql/datacatalog/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/datacatalog/SchemaUtils.java

```diff
@@ -53,7 +53,7 @@
           .build();
 
   /** Convert DataCatalog schema to Beam schema. */
-  static Schema fromDataCatalog(com.google.cloud.datacatalog.v1beta1.Schema dcSchema) {
+  public static Schema fromDataCatalog(com.google.cloud.datacatalog.v1beta1.Schema dcSchema) {
```

Comment:
Rather than making these functions public so our internal code can use them, could we try to move the internal code into Beam? I think it should be possible to move it into DataCatalogTableProvider

Reply:
Done. PTAL.